src/style.css

```diff
@@ -1,3 +1,61 @@
+:root {
+  --link-color: #00ACDC;
+  --header-height: 60px
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  height: var(--header-height);
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.nav_list{
+  padding: 0;
+  display: flex;
+  list-style: none;
+  text-transform: uppercase;
+}
```

Comment:
The class '.nav_list' should be named using BEM notation as '.nav__list' to maintain consistency with BEM naming conventions, which you appear to be using in the rest of your CSS.